main.go

```diff
@@ -461,11 +553,85 @@ func (svr *bridge) handleCall(w http.ResponseWriter, r *http.Request) {
 	http.Error(w, strings.Join(text, "\n"), respCode)
 }
 
+func parseMsgTemplates(tmplPath string, token string) (*ut.Template, error) {
```

Comment:
Also, the `parseMsgTemplates` could be added during startup instead of the handle since the token is not required as part of the call. I had it in the `handleCall` to allow the user templates to be dynamically updated without any restart of the bridge.

Reply:
Aye - thank you for the response. I think moving this into startup would address my primary concern (which I didn't quite explain well). I'm thinking that: as a user of the software, if my templates are bad I would want to know that ASAP. It would be bad if my alert gets lost due to something I could have corrected as I deployed my templates, so moving this error detection early in the startup cycle would help prevent that scenario.

(for reference, my deployment workflow is via Chef where the binary, systemd unit, config files, and service start/restart logic are all managed together - but those running in containers/K8S would also benefit from an init failure as a problem signal)